app/javascript/components/pages/ApartmentIndex.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import apartments from "../components/MockData";
+
+class ApartmentIndex extends Component {
+  render() {
+    let data = apartments;
```

Comment:
I would consider passing this info as props from `App.js` rather than using your mock data. Eventually you will have this data coming from your db.